.github/workflows/builds.yml

```diff
@@ -363,8 +364,21 @@ jobs:
       - uses: actions/checkout@v4
         with:
           path: input-leap
-          submodules: recursive
 
+      - uses: ilammy/msvc-dev-cmd@v1
+      - uses: Bacondish2023/setup-googletest@v1
+
+      - name: Setup MSYS2 with pkg-config
```

Comment:
why do you need msys2 ? our package is using msvc to build

Reply:
Otherwise the Qt5 version doesnt find PkgConfig and fails - is there a better way?